src/commands/Pokemon/pokedex.ts

```diff
@@ -0,0 +1,169 @@
+import { DexDetails, GenderEntry } from '@favware/graphql-pokemon';
+import { toTitleCase } from '@klasa/utils';
+import { MessageEmbed } from 'discord.js';
+import { CommandStore, KlasaMessage } from 'klasa';
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { getPokemonDetailsByFuzzy, GraphQLPokemonResponse, parseBulbapediaURL, POKEMON_EMBED_THUMBNAIL, POKEMON_GRAPHQL_API_URL } from '../../lib/util/Pokedex';
+import { fetch } from '../../lib/util/util';
+
+const BaseStats: Record<string, string> = {
+	hp: 'HP',
+	attack: 'ATK',
+	defense: 'DEF',
+	specialattack: 'SPA',
+	specialdefense: 'SPD',
+	speed: 'SPE'
+};
+
+export default class extends SkyraCommand {
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			aliases: ['pokemon', 'dex'],
+			cooldown: 10,
+			description: language => language.tget('COMMAND_POKEDEX_DESCRIPTION'),
+			extendedHelp: language => language.tget('COMMAND_POKEDEX_EXTENDED'),
+			requiredPermissions: ['EMBED_LINKS'],
+			usage: '<pokemon:pokemon>',
+			flagSupport: true
+		});
+
+		this.createCustomResolver('pokemon', arg => arg.toLowerCase());
```

Comment:
No need to create a custom resolver, just cast it to lower case in the run method